shared_files/config/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!*.encrypted
```

Comment:
I didn't know you could use `!` in `.gitignore` 💯 
Does this mean that all files except `README.md` and `*.encrypted` are ignored by `git`?

Reply:
Yes!  I would have preferred to keep all the settings in the .gitignore at the root of the repo, but this strategy works best in its own file.